src/core/ucc_context.c

```diff
@@ -275,8 +271,12 @@ ucc_status_t ucc_context_create(ucc_lib_h lib,
     ctx->lib = lib;
     ucc_copy_context_params(&ctx->params, params);
     ucc_copy_context_params(&b_params.params, params);
-    b_params.context = ctx;
-    status = ucc_create_tl_contexts(ctx, config);
+    b_params.context           = ctx;
+    b_params.estimated_num_eps = 0; //TODO
```

Comment:
Eventually, how do you expect it to be filled?

Reply:
We can:
* get it from runtime via context_config_modify (first and simplest solution)
* compute inside libucc using oob if provided (e.g., if we build local info for hierarchies, it will be there)